src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.h

```diff
@@ -125,28 +134,56 @@ class Delegate
     virtual ~Delegate() = default;
 
     /**
-     *   @brief Handle Command Callback in application: SetCookingParameters
-     *   @return Returns the Interaction Model status code which was user determined in the business logic
-     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level
-     *   @param  cookTime: the input cook time value
-     *   @param  powerSetting: the input power setting value
+     *   @brief Handle Command Callback in application: SetCookingParameters.
+     *   @return Returns the Interaction Model status code which was user determined in the business logic.
+     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level.
```

Comment:
I don't understand the documentation for this param....  Isn't cookMode just the value that should be used from the cook mode?  I don't know what "which from the Microwave Oven Mode application level" means.